src/components/blocks/software/Code/index.js

```diff
@@ -89,10 +89,7 @@ const Code = ({ data, attribs }) => {
   const contentWithObfuscatedKey = useMemo(
     () =>
       dataContainsKey
-        ? contentWithRandomChannelName.replace(
-            /{{API_KEY}}/g,
-            '*********************************************************',
-          )
+        ? contentWithRandomChannelName.replace(/{{API_KEY}}/g, new Array(57).join('*'))
```

Comment:
Sorry, last one, that 57 needs to be a constant. Seems like an arbitrary number, so good luck naming that!